src/sorcha/sorcha.py

```diff
@@ -189,7 +162,12 @@ def runLSSTPostProcessing(cmd_args, configs, pplogger=None):
         verboselog("Working on objects {}-{}.".format(startChunk, endChunk))
 
         # Processing begins, all processing is done for chunks
-        observations = reader.read_block(block_size=configs["size_serial_chunk"])
+        if configs["ephemerides_type"].casefold() == "external":
```

Comment:
This is the logic that will either read from an external ephemeris file, or pass in a chunk of the orbits dataframe for A&R to process.